### PR TITLE
SNOW-196978 add is_old_driver to the Jenkins job

### PR DESCRIPTION
**Pull request by sfc-gh-stakeda**
This is the second half of changes for Old driver test for Jenkins.

### CODE REVIEW

**Comment by sfc-gh-stakeda** [comment]
https://jenkins.int.snowflakecomputing.com/job/RT-PyConnector37-OldDriver/

File: ci/test_linux.sh [hunk 1]
```diff
@@ -28,12 +28,18 @@ mv ${CONNECTOR_DIR}/test/parameters_jenkins.py ${CONNECTOR_DIR}/test/parameters.
...
+        TEST_ENVLIST=fix_lint,py${SHORT_VERSION}-{extras,unit,integ,pandas,sso}-ci,coverage
+        echo "[Info] Running tox for ${TEST_ENVLIST}"
 
-    python3 -m tox -e ${TEST_ENVLIST} --external_wheels ${CONNECTOR_WHL}
-done
+        python3 -m tox -e ${TEST_ENVLIST} --external_wheels ${CONNECTOR_WHL}
+    done
+else
+    # Old Driver Test
+    echo "[Info] Testing with ${PYTHON_VERSION}"
```
Comment (sfc-gh-mkeller) [inline ci/test_linux.sh:43]: Please not that `PYTHON_VERSION` is not defined, `PYTHON_VERSIONS` is.
How about?
```suggestion
    echo "[Info] Running old connector tests"
```

Comment (sfc-gh-stakeda) [reply ci/test_linux.sh]: fixed

**Review by sfc-gh-mkeller — APPROVED** [review]
🚢 🚢

**Pull request merged**